packages/toc-extension/schema/plugin.json

```
@@ -0,0 +1,14 @@
+{
+  "title": "Table of Contents",
+  "description": "Table of contents settings.",
+  "properties": {
+    "numberingH1": {
+      "title": "Enable h1 numbering",
+      "description": "Specifies whether to number first level headers",
```

```suggestion
      "description": "Whether to number first level headings",
```

These are called headings, headers are a different thing (https://html.spec.whatwg.org/multipage/sections.html)